app/components/UserAvatar.module.css

```
@@ -20,10 +23,10 @@
     }
 }
 
-@media (--sm) {
+@media (--tablet) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)